### PR TITLE
RPRBLND-1968: Improve MatX UI panel (should be more close to Blender Shading)

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Improve MatX UI panel (should be more close to Blender Shading).

### EFFECT OF CHANGE
Improved MatX UI panel.
Added UI element to animate property.
Fixed behavior Displacement output link/remove a node to Surface output.

### TECHNICAL STEPS
Made changes to MxNodeInputSocket.draw, so socket property has a different appearance according to context.
Redesign layout of the Material Properties panel, stuff like proportions, icons, position.
Changed values of  use_property_split, use_property_decorate in a few places to add animation UI items.
Fixed Displacement output link/remove node, pass to HDUSD_MATERIAL_OP_invoke_popup_shader_nodes correct input number input_num = int(input.identifier[-1]).

### CODE REVIEW

File: src/hdusd/mx_nodes/nodes/base_node.py [hunk 1]
```diff
@@ -152,11 +156,19 @@ def draw_buttons(self, context, layout):
         nodedef = self.nodedef
 
         if self._ui_folders:
-            col = layout.column(align=True)
+            new_layout = layout
+            if context.area.type == "PROPERTIES":
+                new_layout = layout.split(factor=0.38)
```
Comment (bnagirniak) [inline src/hdusd/mx_nodes/nodes/base_node.py:161]: factor=0.4 is standard

**Review by bnagirniak — CHANGES_REQUESTED** [review]
Some UI places should be improved, discussed offline

**Review by DagerD — CHANGES_REQUESTED** [review]
Tested. Works ok. Some improvements were discussed offline

**Review by bnagirniak — APPROVED** [review]
Tested - looks and works good.

**Review by DagerD — APPROVED** [review]
Tested. Works ok.

**Review by bsavery — CHANGES_REQUESTED** [review]
Merge error

**Pull request merged**